Src/Plugins/Security/Security.SharedStrings.txt

```diff
@@ -58,7 +58,7 @@
   $SEC101/022.PayPalBraintreeAccessToken=access_token\$production\$(?P<secret>[0-9a-z]{16}\$[0-9a-f]{32})
   $SEC101/023.AmazonMwsAuthToken=(?:[^a]|^)(?P<secret>amzn\\.mws\\.[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})(?:[^0-9a-f]|$)
   $SEC101/024.TwilioApiKey=(?:[^S]|^)(?P<secret>SK(?i)[0-9a-f]{32})(?:[^0-9a-f]|$)
-  $SEC101/028.PlaintextPassword=(?i)ConvertTo-SecureString\s*(?:-[S]tring)?\s*["'](?P<secret>[^"')(?:]*)["']
+  $SEC101/028.PlaintextPassword=(?i)ConvertTo-SecureString\s*(?:-[S]tring)?\s*["']*(?P<secret>[^"')(?:]*)["']*
```

Comment:
This looks good. Once the tests are passing and there's an update to the release history, I'm happy to approve this change. Can we get SEC101/001 disabled as well? The manual changes to the .json files are not optimal. I'm okay with that being a separate PR.

Reply:
@marmegh Added the description in `ReleaseHistory`. Fixed failing tests. 
@HulonJenkins is creating a separate PR for `SEC101/001`.